modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -752,10 +762,14 @@ private CompletableFuture<List<Assignments>> writeTableAssignmentsToMetastore(
                             });
                         }
                     })
-                    .exceptionally(e -> {
-                        LOG.error("Couldn't write assignments to metastore", e);
-
-                        return null;
+                    .handle((realAssignments, e) -> {
+                        if (e != null) {
+                            LOG.error("Couldn't write assignments {} to metastore during invoke",
```

Comment:
from my point of view, the message is a bit incorrect, exception could caught here not only from invoke, let's rephrase it